.tarantoolctl

```diff
@@ -0,0 +1,16 @@
+-- Options for test-run tarantoolctl
+
+local workdir = os.getenv('TEST_WORKDIR')
+default_cfg = {
+    pid_file   = workdir,
+    wal_dir    = workdir,
+    memtx_dir  = workdir,
+    vinyl_dir  = workdir,
+    log        = workdir,
+    background = false,
+    replication_sync_timeout = 100,
```

Comment:
Should be passed from the option (via the environment variable).

Reply:
Set to use environment variable.